lib/babe/verify_integration_test.go

```diff
@@ -374,9 +368,7 @@ func TestVerifyPimarySlotWinner(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	ok, err := verifier.verifyPrimarySlotWinner(
-		babeHeader.AuthorityIndex, slot.number,
-		babeHeader.VRFOutput, babeHeader.VRFProof)
+	ok, err := verifier.verifyPrimarySlotWinner(authorityIndex, slotNumber, vrfOutput, vrfProof)
```

Comment:
Why not use directly fields of `d` for args

```suggestion
	ok, err := verifier.verifyPrimarySlotWinner(d.AuthorityIndex, slotNumber, d.VRFOutput, d.VRFProof)
```

And remove the variable assignments above

Reply:
adressed